README.md

```diff
@@ -12,21 +12,45 @@ Setting up a fully working Linux system with all needed packages and openHAB rec
 
-Closely related to openHABian is the repository [openhab-linuxpkg](https://github.com/openhab/openhab-linuxpkg) which provides linux packages for openHAB. openHABian uses those packages to setup openHAB with some additional configurations.
+## Hardware and OS support
+As of openHABian version 1.5.1 of the image, all Raspberry Pi models are supported as hardware.
+Anything x86 based may work or not. Anything else ARM based such as ODroids, OrangePis and the like may work or not.
+NAS servers such as QNAP and Synology boxes will not work. Support for PINEA64 was also dropped in this current release.
+We strongly recommend to choose Raspberry Pi 2, 3 or 4 systems to have 1GB of RAM or more. RPi 1 and 0/0W only have a single CPU core and 512MB. This can be sufficient to run a smallish openHAB setup, but it will not be enough to run a full-blown system with many bindings and memory consuming openHABian features/components such as ZRAM, InfluxDB or Grafana.
+We do not actively prohibit installation on any hardware - including unsupported systems -, but we might skip or deny to install specific extensions such as those memory hungry features named above.
+Supporting a hardware means to test every single patch and every release. There's simply too many SBC and HW combinations we don't have available or the time to spend on testing efforts it takes to make openHABian a reliable system.
+Let's make sure you understand the implications of these statements: it means that to run hardware other than RPi 2/3/4 or x86 is **not** _fully_ supported.
+You can try and there's a good chance it'll work out for you. But if you hit any problem related to memory or hardware, you'll be on your own. You are expected **not** to raise these problems as issues on the community forum or on GitHub.
 
-## Installation and Setup
+Going beyond what the RPi image provides, as a manually installed set of scripts, openHABian is supposed to run on all Debian like Linux distributions on either ARM or x86 hardware. Note our automated testing for x86 is based on generic Debian and we take no resposibilities if you run into trouble with a different Linux flavour.
+We recommend to use Ubuntu for x86 hardware and ARMbian for ARM hardware that we don't support.
+Note we expect you to use an up to date distribution: 'Buster' for Raspbian and 'Bionic Beaver' for Ubuntu, these are. To install openHABian on anything older may work or not.
+
+Either way, please note that you're on your own when it comes to configuring and installing the HW with the proper OS yourself.
+
+### 64 bit ?
+Although RPi3 and 4 have a 64 bit processor, you cannot run openHAB in 64 bit.
+The Azul Java Virtual Machine we currently use is incompatible with the aarch64 ARM architecture.
+In general you should be aware that to run in 64 bit has a major drawback, doubled memory usage that essentially is.
+Not quite a good idea on a heavily memory constrained platform like a RPi.
```

Comment:
Combine these statements "In general you should be aware that to run in 64-bits has a major drawback, doubled memory usage, that is not a good idea on a heavily memory constrained platform like a RPi.

Reply:
ok